components/protocol-wasm/src/lib.rs

```diff
@@ -0,0 +1,213 @@
+use chrono::{DateTime, Utc};
+use base64::{Engine as _, engine::{self, general_purpose}};
+use std::str::FromStr; // 0.4.15
+
+use bindings::protocol;
```

Comment:
maybe alias `bindings::protocol` to something (just `bindings` maybe?) so it's clearer that it comes from the component bindings and doesn't refer to the protocol crate itself.

Reply:
How about `use bindings::protocol as protocolbindings`?  I don't think I can use `bindings` as wit-bindgen uses that as a keyword in an export at the bottom, and I also run into issues using `-` and `_` in the alias name

Reply:
Went ahead and did this, happy to rename if you think of something better